ingestion/strip_data.py

```diff
@@ -0,0 +1,126 @@
+"""This script strips the raw data to only include the columns we need for the project."""
+
+import pandas as pd
+import logging
+import sys
+
+def austin() -> pd.DataFrame:
+    """
+    Extracts needed columns from the Austin data and saves it as a csv file
+    :return austin_stripped: the stripped data as a pandas dataframe
+    """
+    AUSTIN_RAW_PATH = "./raw_data/austin.csv"
+    AUSTIN_STRIPED_PATH = "./stripped_data/austin.csv"
+    austin_raw = pd.read_csv(AUSTIN_RAW_PATH)
+    austin_stripped = austin_raw[austin_raw.contractor_trade == "Electrical Contractor"]
+    austin_stripped = austin_stripped[["issue_date", "location", "contractor_company_name"]]
+    # {'latitude': '30.29406429', 'longitude': '-97.69323996', 'human_address': '{""address"": """", ""city"": """", ""state"": """", ""zip"": """"}'}
+    austin_stripped["latitude"] = austin_stripped["location"].apply(lambda x: x.split(",")[0].split("'")[3] if type(x) == str else "nan")
+    austin_stripped["longitude"] = austin_stripped["location"].apply(lambda x: x.split(",")[1].split("'")[3] if type(x) == str else "nan")
```

Comment:
I think using "nan" will replace with the string nan but you will want to use numpy.NaN